FirebaseAuth/Sources/MultiFactor/TOTP/FIRTOTPSecret.m

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import <TargetConditionals.h>
+#if TARGET_OS_IOS
+
+#import <UIKit/UIKit.h>
+#import "FirebaseAuth/Sources/MultiFactor/TOTP/FIRTOTPSecret+Internal.h"
+#import "FirebaseAuth/Sources/Public/FirebaseAuth/FIRTOTPSecret.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation FIRTOTPSecret
+
+// explicity synthesize
+@synthesize secretKey = _secretKey;
```

Comment:
Question - why do we need synthesize annotation?

Reply:
Thanks for catching this. It was initially address some build failures. It seems to be working now since the code is better integrated.